### ISSUE TITLE
Should `ResolveSpanStatusForHttpStatusCode's` implementation be aligned between this repo and the contrib one?

### ISSUE THREAD

**Issue by joegoldman2** [open-telemetry/opentelemetry-dotnet#5347]
Today, the implementation of `ResolveSpanStatusForHttpStatusCode` is different between this repo and the contrib one.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/303ab7a25a81d7804ac7414134366e61a9501114/src/Shared/SpanHelper.cs#L20-L29

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/89ff089bf07fe8576e7cf2027a020a305537ad91/src/Shared/SpanHelper.cs#L41-L50

 Should the two implementations be aligned?

**Comment by vishweshbankwar** [comment]
@joegoldman2 Yes - would you be interested in sending the PR to contrib repo?

**Comment by joegoldman2** [comment]
Will do!
